drivers/lis2dh12/lis2dh12.c

```diff
@@ -187,6 +194,72 @@ int lis2dh12_read(const lis2dh12_t *dev, int16_t *data)
     return LIS2DH12_OK;
 }
 
+int lis2dh12_set_int(const lis2dh12_t *dev, const lis2dh12_int_params_t *params, uint8_t int_line)
+{
+    gpio_t pin = dev->p->int_pin[int_line - 1];
+
+    assert(dev && params->int_config && params->int_type);
+
+    assert (params->int_threshold >= 0);
+    assert (params->int_duration >= 0);
+
+    assert (pin != GPIO_UNDEF);
+
+    if (gpio_init_int(pin, GPIO_IN, GPIO_RISING, params->cb, params->arg)) {
+        return LIS2DH12_NOINT;
+    }
+
+    _acquire(dev);
+
+    switch (int_line){
+        /* first interrupt line (INT1) */
+        case 1:
```

Comment:
Please define an enumeration like
```
enum {
    LIS2DH12_INT1 = 1;
    LIS2DH12_INT2 = 2;
};
```
and use
```suggestion
        case LIS2DH12_INT1:
```
It is more name giving and easier to use.